### ISSUE TITLE
Check for file attachments in e-mail

### ISSUE THREAD

**Issue by schmunk42** [captbaritone/codeception-mailcatcher-module#34]
Is there a method to check for a specific file attachment name?

**Comment by schmunk42** [comment]
Found a way :)

```
$I->seeInLastEmailTo($this->email, 'Content-Disposition: attachment; filename=profile.pdf');
```

Would be nice to have a helper function for that.

**Comment by gdscei** [comment]
Great suggestion! If you could include a PR, otherwise I will get this done as soon as I can.

**Comment by EstebanSerfe** [comment]
@schmunk42 Check #37

**Comment by Jamesking56** [comment]
Implemented by #70
